module/module.go

```diff
@@ -72,7 +72,7 @@ func (h HandlerMap) ToProto() *pb.HandlerMap {
 // NewHandlerMapFromProto converts protobuf to HandlerMap.
 func NewHandlerMapFromProto(ctx context.Context, pMap *pb.HandlerMap, conn *grpc.ClientConn) (HandlerMap, error) {
 	hMap := make(HandlerMap)
-	refClient := grpcreflect.NewClient(ctx, reflectpb.NewServerReflectionClient(conn))
+	refClient := grpcreflect.NewClientV1Alpha(ctx, reflectpb.NewServerReflectionClient(conn))
```

Comment:
I changed this as the new version of `relectpb` outputs this lint warning:
```module/module.go:75:15: SA1019: grpcreflect.NewClient is deprecated: Use NewClientV1Alpha if you are intentionally pinning the v1alpha version of the reflection service. Otherwise, use NewClientAuto instead. (staticcheck)
        refClient := grpcreflect.NewClient(ctx, reflectpb.NewServerReflectionClient(conn))
                     ^
cli/client.go:593:15: SA1019: grpcreflect.NewClient is deprecated: Use NewClientV1Alpha if you are intentionally pinning the v1alpha version of the reflection service. Otherwise, use NewClientAuto instead. (staticcheck)
        refClient := grpcreflect.NewClient(refCtx, reflectpb.NewServerReflectionClient(conn))
                     ^
robot/client/client.go:328:15: SA1019: grpcreflect.NewClient is deprecated: Use NewClientV1Alpha if you are intentionally pinning the v1alpha version of the reflection service. Otherwise, use NewClientAuto instead. (staticcheck)
        refClient := grpcreflect.NewClient(rc.backgroundCtx, reflectpb.NewServerReflectionClient(conn))
                     ^
INFO File cache stats: 38 entries of total size 393.0KiB
INFO Memory: 36 samples, avg is 127.6MB, max is 443.1MB
INFO Execution took 3.499068041s
make: *** [lint-go] Error 1
```

I'm not sure if I should go to the new api version or pin to the existing behavior as, I am unfamiliar with this code so I did the latter to reduce risk of breaking things.